mitiq/tests/test_without_third_party_packages.py

```diff
@@ -20,14 +22,14 @@ def test_import():
     """
     import mitiq
 
-    if isinstance(mitiq.QPROGRAM, ABCMeta):
-        pass  # If only Cirq is installed, QPROGRAM is not a typing.Union.
-    else:
+    if get_origin(mitiq.QPROGRAM) is Union:
```

Comment:
Nice, this is an easier way to understand that the type of something is a `Union`!